courses/machine_learning/deepdive2/image_classification/labs/2_mnist_models.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Change first link from corp to public version.</p><p><a href="http://console.cloud.google.com/gcr" target="_blank">http://console.cloud.google.com/gcr</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>